astro/src/styles/bootstrap.scss

```diff
@@ -165,37 +182,77 @@ $utilities: map-merge(
 }
 
 .hero-link:focus-visible, .hero-button:focus-visible {
-  outline-color: white;
+  outline-color: #FFFFFF;
 }
 
 .form-field:focus {
   border-style: solid;
 }
 
+.breadcrumb-item, .breadcrumb-item.active, .nav-link, .dropdown-item, .dropdown-item:hover {
+  color: #000000;
+}
+
+.dropdown-item:hover {
+  background-color: $body-tertiary-bg;
+}
+
 .navbar-brand, .text-brand {
   font-family: $headings-font-family;
 }
 
+.card, .card-header {
+  background-color: $body-secondary-bg;
```

Comment:
The blocks of white of the cards are really harsh in dark mode. Dark mode is meant to reduce the harshness of block of light colors. Perhaps this works when if we shade the secondary color darker.

The text color is also white for those cards, so it's also completely illegible.